### PR TITLE
added object template

### PR DESCRIPTION
**Pull request by Sekhar-Kumar-Dash**
this PR closes:https://github.com/keploy/keploy/issues/419

### CODE REVIEW

**Review by nehagup — COMMENTED** [review]
@Sekhar-Kumar-Dash the issue is regarding adding some sample requests for the user-profile application in the readme so that one trying the sample app can copy the sample request given and quickly see the expected results.

**Comment by Sekhar-Kumar-Dash** [comment]
> 

@nehagup so you mean i have to add sample like this 
Screenshot (130)
if yes how many would be good

**Comment by nehagup** [comment]
@Sekhar-Kumar-Dash no, I mean something like this example - https://github.com/keploy/keploy#1-generate-shortened-url

**Comment by Sekhar-Kumar-Dash** [comment]
> @Sekhar-Kumar-Dash no, I mean something like this example - https://github.com/keploy/keploy#1-generate-shortened-url

@nehagup  i made the changes please review it again

File: users-profile/README.md [hunk 1]
```diff
@@ -77,6 +78,19 @@ go run .
 > Keploy Port: http://localhost:6789/testlist
 
 To generate Test Cases, you need to make some API calls. It could be using Thunder Client, Postman Desktop Agent, or your preferred API testing tool.
+here you can use this object template below for testing:
+```shell
+you can copy paste this object template:
+{
```
Comment (nehagup) [inline users-profile/README.md:84]: @Sekhar-Kumar-Dash this needs to be changed to a valid curl request.

Comment (Sekhar-Kumar-Dash) [reply users-profile/README.md]: @nehagup  I made the changes please review it again

**Pull request merged**